GitCommands/Git/GitModule.cs

```diff
@@ -2129,10 +2129,18 @@ public IReadOnlyList<string> GetRemoteNames()
 
         public async Task<IReadOnlyList<Remote>> GetRemotesAsync()
         {
-            return ParseRemotes(await _gitExecutable.GetOutputLinesAsync("remote -v"));
+            ExecutionResult result = await _gitExecutable.ExecuteAsync("remote -v");
+            ////TODO: Handle non-empty result.StandardError if not result.ExitedSuccessfully
+            return result.ExitedSuccessfully
+                ? ParseRemotes(result)
+                : new List<Remote>();
 
-            IReadOnlyList<Remote> ParseRemotes(IEnumerable<string> lines)
+            static IReadOnlyList<Remote> ParseRemotes(ExecutionResult result)
             {
+                IEnumerable<string> lines = result.StandardOutput
+                    .LazySplit('\n', StringSplitOptions.RemoveEmptyEntries)
+                    .Concat(result.StandardError
+                    .LazySplit('\n', StringSplitOptions.RemoveEmptyEntries));
```

Comment:
Please fix the indentation to reflect the nesting

Reply:
I have joined the `LazySplit`s with the previous lines.

Reply:
Is not the unofficial to have all one one line or one line per argument?
The indentation is not automatic though...